modules/hitlnext/src/views/full/app/components/ConversationContainer.tsx

```diff
@@ -24,7 +24,7 @@ const ConversationContainer: FC<Props> = ({ api, bp }) => {
   async function handleAssign() {
     try {
       const handoff = await api.assignHandoff(state.selectedHandoffId)
-      toast.success(lang.tr('module.hitlnext.handoff.assigned', { id: handoff.id }))
+      toast.success(lang.tr('module.hitlnext.handoff.assigned', { id: handoff.id }) ,'' ,{ toasterProps: {position: Position.TOP}})
```

Comment:
As mentioned by Yann, there are missing spaces. Make sure prettier runs on file save.

```suggestion
      toast.success(lang.tr('module.hitlnext.handoff.assigned', { id: handoff.id }) , '' , { toasterProps: { position: Position.TOP }})
```